tests/integration/test_wild.py

```diff
@@ -40,11 +40,13 @@ def _pretend_to_be_flickr_library():
         data = r.read()
         h.close()
 
-    with vcr.use_cassette('fixtures/vcr_cassettes/flickr.yaml') as cass:
+    c_file = str(tmpdir.join('fixtures/vcr_cassettes/flicr.yaml'))
```

Comment:
`flicr.yaml` -> `flickr.yaml`

and it looks like this is a separate bugfix, which could be submitted separately.